README.md

```diff
@@ -70,19 +70,19 @@ Ask for help or report issues:
 
 ### Build Stride
 
-1. Open a command prompt, point it to a directory and clone Stride to it: `git lfs clone https://github.com/stride3d/stride.git`
-   - Note that when you use GitHub -> Code -> Download ZIP, this doesn't support Large File Support ```lfs```, make sure you use the command above or that your git client does it for you
+1. Clone the repo with a git UI (recommended) or open a command prompt, point it to a directory and clone Stride to it: `git lfs clone https://github.com/stride3d/stride.git`. **Do NOT use GitHub -> Code -> Download ZIP**, this doesn't include the ```lfs``` files.
 2. Open `<StrideDir>\build\Stride.sln` with Visual Studio 2022 and build `Stride.GameStudio` in the 60-Editor solution folder (it should be the default startup project) or run it from VS's toolbar.
    - Optionally, open and build `Stride.Android.sln`, `Stride.iOS.sln`, etc.
 
 #### Build Stride without Visual Studio
 
 1. Install [Visual Studio Build Tools](https://aka.ms/vs/17/release/vs_BuildTools.exe) with the same prerequisites listed above
 2. Add MSBuild's directory to your system's *PATH* (ex: `C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin`)
-3. Open a command prompt, point it to a directory and clone Stride to it: `git clone https://github.com/stride3d/stride.git`
+3. Clone the repo with a git UI (recommended) or open a command prompt, point it to a directory and clone Stride to it: `git clone lfs https://github.com/stride3d/stride.git`
 4. Navigate to `/Build` with the command prompt, input `msbuild /t:Restore Stride.sln` then `compile.bat`
 
 If building failed:
+* Some errors for test projects are normal if you haven't setup the testing evironment, GameStudio will start fine anyway.
```

Comment:
Do we have instructions how to set up testing environment?

Reply:
I don't think there's anything special for building test projects. The one thing I remember that might fall to build is the VS extension, because it needs an additional SDK installed.